### ISSUE TITLE
"PURPOSE not defined" on boot loader

### ISSUE THREAD

**Issue by jimbygh** [project-trident/trident-installer#13]
The cause is incorrect formatting of file /boot/logo-trident.4th read by the forth interpreter.
The words PURPOSE, CONSEQUENTIAL, STRICT, and WAY,  all part of the standard legal disclaimer, are incorrectly formatted (looks like bad cut and paste) are are interpreted by forth.  This causes an error on processing of logo-trident.4th and no trident is displayed.

However, if corrected, a separate problem occurs.  The file "logo-trident.4th" is displayed resulting in an overlapping trident on top of the displayed menu (brand-trident.4th), as shown in the attached image.
![20180904_171904 2](https://user-images.githubusercontent.com/42945727/45059438-fa2b5a80-b069-11e8-9f7a-e97e7fca7f47.jpg)

A horizontal trident is suggested for

**Comment by jimbygh** [comment]
A horizontal trident is suggested for logo-trident.4th.

**Comment by jimbygh** [comment]
Here's a sample of a horizontal trident for an artist better than me :-)


[logo-trident.4th.txt](https://github.com/project-trident/trident-installer/files/2350673/logo-trident.4th.txt)

File must be installed as /boot/logo-trident.4th  to work.

**Comment by beanpole135** [comment]
Thank you for pointing that out!
I just fixed the brand-trident.4th file in this commit: https://github.com/trueos/trueos/commit/f69c78502b08ddd0583faae38bc8d2a185cf7b60#diff-6883b5ddd892906d6e5a15efeff52ace
and I fixed the logo-trident.4th file for the syntax issues in this commit: https://github.com/trueos/trueos/commit/7d23d5a2e0e288567136b33c992487f144fabab7#diff-6883b5ddd892906d6e5a15efeff52ace

The "logo" will always appear on the right-side of the menu box for the FreeBSD boot menu, so a vertical trident seems to make more sense so that the boot menu can fit on 4:3 aspect ratio screens instead of assuming a 16:9 aspect ratio. The only thing we will still need to test/adjust for the logo file now is probably the placement/scaling of the ascii. I am not terribly familiar with forth, but it seems like we might be able to just adjust line 29 (reproduced below) to move the logo a bit further to the right.
```
52 logoX ! 9 logoY ! \ Initialize logo placement defaults
```
